### DISCUSSION TITLE
how can integrade DB?

### DISCUSSION THREAD

**Discussion by HalloWelt42** [dunglas/symfony-docker#597] [Q&A]
Unfortunately, I understand absolutely nothing about the completely poorly documented setting options

I would also like to switch to Symfony 6.4, because the 7.x is totally buggy, especially bad when migrating under MYSQL/MariaDB

I cannot access the database for postgresql from outside via DATAGrip because all port shares in the env are completely missing. I don't even know which service is running

:(

Translated with DeepL.com (free version)

**Comment by maxhelias** [comment]
> Unfortunately, I understand absolutely nothing about the completely poorly documented setting options

Unfortunately, I don't know what else to add. You can contribute, if you find it completely poorly, it's open source.

> I would also like to switch to Symfony 6.4, because the 7.x is totally buggy, especially bad when migrating under MYSQL/MariaDB

Switch back to Symfony 6.4, it's a composer basic and not related to this project.
Personally, I have no problems with 7.x. Maybe you have doctrine issue and not symfony.

> I cannot access the database for postgresql from outside via DATAGrip because all port shares in the env are completely missing. I don't even know which service is running

It's not because the port is not defined that the service does not work. This is deliberate, so that docker can choose which port to expose. Look at the container list to find out, it's also a docker basic.
